index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - setup and mobile first</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="toolbar">
+            <div class="container-1">
+                <div class="clock"><p><strong>9:41</strong></p></div>
+                <div class="space2"></div>
+                <div class="conn"><img class="conn-im" alt="Connetion" src="images/connection.png"></div>
+                <div class="wire"><img class="wire-im" alt="Wireless" src="images/wireless.png"></div>
+                <div class="batt"><img class="batt-im" alt="Battery" src="images/battery.png"></div>
+            </div>
+            <div class="container-2">
+                <div class="logo"><a href="#home"><img class="mylogo-im" alt="MyLogo" src="images/mylogo.png"></a></div>
+                <div class="space1"></div>
+                <nav class="menu"><div><img class="menu-im" alt="Menu" src="images/menu.png"></div></nav>
+            </div>
+        </div>
+    </header>
```

Comment:
- [ ] As required for the project, kindly make sure your header design sticks as much as possible to the Figma design

| Figma Design | Your Design |
| ------------------ | ------------------- |
| ![header](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/toolbar-mobile.png?raw=true) | ![Screenshot from 2023-03-24 18-15-29](https://user-images.githubusercontent.com/83042742/227653291-04336225-4ce8-4b28-a962-273877a8fb61.png) |